src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.WorkItem.cs

```diff
@@ -99,21 +99,16 @@ public object Key
 
                 private ImmutableHashSet<IIncrementalAnalyzer> Union(ImmutableHashSet<IIncrementalAnalyzer> analyzers)
```

Comment:
Perhaps although it might be too easy to use the overload defined on ```ImmutableHashSet<T>```, especially if the extension method is in a different namespace. Will leave as is for now.